### DISCUSSION TITLE
failed to apply migration _multi_match_migrate

### DISCUSSION THREAD

**Discussion by gladson** [pocketbase/pocketbase#2920] [Q&A]
pocketbase version 0.16.0

+ pocketbase serve --dir=/opt/pocketbase --publicDir=/var/pocketbase/public --http=0.0.0.0:8090
2023/07/14 23:57:29 Failed to apply migration 1673167670_multi_match_migrate.go: SQL logic error: no such column: indexes (1)

**Comment by ganigeorgiev** [comment]
There are not enough details to help you debug it. I assume that you've attempted to update your PocketBase executable, right? What was the previously used version?

Big jumps between 0.X versions may cause the above because the Collection model is still changing.
You can try to perform the executable update on several steps.
Based on the error, you can try first with [v0.13.4](https://github.com/pocketbase/pocketbase/releases/tag/v0.13.4) (_the latest release before the `Indexes` field was introduced_) and after that try again with the newest [v0.16.9](https://github.com/pocketbase/pocketbase/releases/tag/v0.16.9) executable.

**Reply by gladson** [reply]
Strangely removing --dir and --publicDir and works...

pocketbase  | + pocketbase serve '--encryptionEnv=f@e5&hszy_6t*yd+0z0sct2q@m#_c=b95!60(vscfkvq!!2plu' --http=0.0.0.0:8090
pocketbase  | 2023/07/15 17:26:19 Server started at http://0.0.0.0:8090
pocketbase  |  ➜ REST API: http://0.0.0.0:8090/api/
pocketbase  |  ➜ Admin UI: http://0.0.0.0:8090/_/

**Reply by ganigeorgiev** [reply]
If you don't set an explicit `--dir`, PocketBase will create a **new** `pb_data` directory next to the executable.

If you want to preserve your old data, please try to upgrade first to v0.13.4 and then to v0.16.9.

**Reply by gladson** [reply]
Thanks my dear.
